lagotto/init.sls

```diff
@@ -63,6 +65,12 @@ include:
   docker_volume.present:
     - name: {{ app_name }}-railsassets
 
+{{ app_name }}-subscribers-config-file:
+  file.managed:
+    - name: /code/config/subscribers.yml
+    - source: salt://lagotto/conf/code/config/subscribers.yml
+    - template: jinja
```

Comment:
Without stating a user/group in this state, this file will be owned by root, is that ok? Typically our config files are owned by the app user.

Reply:
Also, I think we need an on_change here to ensure that changes to this file trigger our docker containers to restart, since the config is only read when the rails server (or sidekiq) starts.

Reply:
@slimeate I'm going to try to avoid this altogether by using formatted environment variable names instead:
https://github.com/PLOS/lagotto/pull/44/commits/11cedb4aa7d4bbd0e1d14dd2a9d2ce03bbfd1a29

I should be able to add these to `props` and they'll be injected into the env.